__tests__/app.test.js

```diff
@@ -152,4 +152,62 @@ describe('GET /api/articles/:article_id/comments', () => {
 
 	})
 })
+describe('PATCH /api/articles/:article_id', () => {
+	it('Should be able to PATCH an article to change the value of its votes and respond with that article and the way it looks after', () => {
+		const patchObj = { inc_votes: 3}
+		return request(app)
+		.patch('/api/articles/1')
+		.send(patchObj)
+		.expect(200)
+		.then( (response) => {
+			expect(response.body.article.votes).toBe(103)
+			expect(response.body.article).toEqual(expect.objectContaining(
+				{
+					article_id: expect.any(Number),
+					title: expect.any(String),
+					author: expect.any(String),
+					body: expect.any(String),
+					created_at: expect.any(String),
+					votes: expect.any(Number),
+				}
+			))
+
+		})
+	})
+	it('Should not allow the user to increment the votes of an article that isnt there', () => {
+		const patchObj = { inc_votes: 3}
+		return request(app)
+		.patch('/api/articles/4444333')
+		.send(patchObj)
+		.expect(404)
+		})
+	it('Should not allow the user to increment the votes of an article that isnt there (sad path edition)', () => {
+		const patchObj = { inc_votes: 3}
+		return request(app)
+		.patch('/api/articles/SELECT*FROM')
+		.send(patchObj)
+		.expect(400)
+		})
+	it('Should not allow the user to increment the votes if the votes are the wrong type of data', () => {
+		const patchObj = { inc_votes: 'chickens'}
+		return request(app)
+		.patch('/api/articles/1')
+		.send(patchObj)
+		.expect(400)
+		})
+	it('Shouldn\'t allow the user to patch a malformed object', () => {
+		const patchObj = { inc_votes: 'SELECT * FROM'}
```

Comment:
this test is technically the same as the one before it - a wrong data type for the vote number 
it's probably better for this test to look at if the `inc_votes` property is missing

Reply:
nvm you've done that in the next test